audb/core/load.py

```diff
@@ -845,3 +885,118 @@ def load_header(
             db.save(db_root_tmp, header_only=True)
             _move_file(db_root_tmp, db_root, define.HEADER_FILE)
     return audformat.Database.load(db_root, load_data=False), backend
+
+
+def load_media(
+        name: str,
+        media: typing.Union[str, typing.Sequence[str]],
+        *,
+        version: str = None,
+        bit_depth: int = None,
+        channels: typing.Union[int, typing.Sequence[int]] = None,
+        format: str = None,
+        mixdown: bool = False,
+        sampling_rate: int = None,
+        cache_root: str = None,
+        num_workers: typing.Optional[int] = 1,
+        verbose: bool = True,
+) -> typing.List:
+    r"""Load media file(s).
+
+    If you are only interested in media files
+    and not the corresponding tables,
+    you should use :func:`audb.load_media`
+    instead of :func:`audb.load`
+    as it is faster.
+
+    Args:
+        name: name of database
+        media: load media files matching the regular expression
```

Comment:
According to https://github.com/audeering/audb/pull/79#issuecomment-832608592 we do not support regular expressions here

Reply:
Indeed, changed it to:
```
media: load media files provided in the list
```

Reply:
Cool, looks good now. I will merge :)